examples/pytorch/CaptumExample/Titanic_Captum_Interpret.py

```diff
@@ -0,0 +1,394 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+"""
+Getting started with Captum - Titanic Data Analysis
+"""
+# Initial imports
+import numpy as np
+import torch
+from captum.attr import IntegratedGradients
+from captum.attr import LayerConductance
+from captum.attr import NeuronConductance
+import matplotlib.pyplot as plt
+from scipy import stats
+import pandas as pd
+import mlflow
+from prettytable import PrettyTable
+from mlflow.utils.autologging_utils import try_mlflow_log
+import tempfile
+import shutil
+import os
+from argparse import ArgumentParser
+import torch.nn as nn
+from torchtext.utils import download_from_url
+
+mlflow.start_run(run_name="Titanic_Captum_mlflow")
+
+
+def TitanicDataset():
+    """
+    we now preprocess the data by converting some categorical features such as
+    gender, location of embarcation, and passenger class into one-hot encodings
+    We also remove some features that are more difficult to analyze
+    After processing, the features we have are:
+    Age: Passenger Age
+    Sibsp: Number of Siblings / Spouses Aboard
+    Parch: Number of Parents / Children Aboard
+    Fare: Fare Amount Paid in British Pounds
+    Female: Binary variable indicating whether passenger is female
+    Male: Binary variable indicating whether passenger is male
+    EmbarkC : Binary var indicating whether passenger embarked @ Cherbourg
+    EmbarkQ : Binary var indicating whether passenger embarked @ Queenstown
+    EmbarkS : Binary var indicating whether passenger embarked @ Southampton
+    Class1 : Binary var indicating whether passenger was in first class
+    Class2 : Binary var indicating whether passenger was in second class
+    Class3 : Binary var indicating whether passenger was in third class
+    url = "https://biostat.app.vumc.org/wiki/pub/Main/DataSets/titanic3.csv"
+    """
+    url = "https://biostat.app.vumc.org/wiki/pub/Main/DataSets/titanic3.csv"
+    download_from_url(url, root="data")
+    dataset_path = "data/titanic3.csv"
+    titanic_data = pd.read_csv(dataset_path)
```

Comment:
```suggestion
    titanic_data = pd.read_csv(url)
```

Can we direcly pass the url to `pd.read_csv` here?

Reply:
@harupy Thanks for suggesting this, yeah we can use this,  I have updated this in the code.